stock_picking_partner_note/models/stock_picking.py

```diff
@@ -10,7 +10,7 @@ class StockPicking(models.Model):
 
     note = fields.Html(compute="_compute_note", store=True)
 
-    @api.depends("partner_id")
+    @api.depends("partner_id", "origin")
```

Comment:
origin is not used in the compute, no reason to add it. It's purely based on partner and picking type.
```suggestion
    @api.depends("partner_id", "picking_type_id")
```

Reply:
The idea behind this depends is to update note when picking is merged (origin is the concatenation of all orders)

Reply:
I got it but the partner is not changing, so the note will remain the same no matter how many SOs you have in the delivery

Reply:
That's the point notes can change in the partner.
Use case:
Partner1 note1 added: Take care with the cat
Sale1 confirmed and note 1 propagated to picking1
Partner1 note2 added: Take care with the dog
Sale2 confirmed and note 1 + note2 should be propagated to picking1

Reply:
Then it should update even without the SO2.
What about a hook on partner for any change of `stock_picking_note_ids`, find all not done/cancel and not printed stock.picking and call `_compute_note` to update the note.

Reply:
> Then it should update even without the SO2. What about a hook on partner for any change of `stock_picking_note_ids`, find all not done/cancel and not printed stock.picking and call `_compute_note` to update the note.

And when do you trigger _compute_note in this case ? Onchange at partner level ?

Reply:
something like this
```
_inherit = "res.partner"
def write(self, vals):
  if `stock_picking_note_ids` in vals:
     open_partner_picking._compute_note()
```

Reply:
That way is an alternative. I'll implement like that

Reply:
@jbaudoux propagating notes from partner to open pickings